Formula/emqx.rb

```diff
@@ -0,0 +1,47 @@
+class Emqx < Formula
+  desc "MQTT broker for IoT"
+  homepage "https://www.emqx.io/"
+  url "https://github.com/emqx/emqx/archive/refs/tags/v5.0.11.tar.gz"
+  sha256 "5b797f768961abccf9ea0050613c02b7d1fc1d3877306d592e72c64a8d95588b"
+  license "Apache-2.0"
+  head "https://github.com/emqx/emqx.git", branch: "master"
+
+  depends_on "autoconf"  => :build
+  depends_on "automake"  => :build
+  depends_on "cmake"     => :build
+  depends_on "coreutils" => :build
+  # emqx cannot be built with Erlang/OTP 25 as of v5.0.11,
+  # but the support is in the works and is coming soon
+  depends_on "erlang@24" => :build
+  depends_on "freetds"   => :build
+  depends_on "libtool"   => :build
+  depends_on "openssl@3"
```

Comment:
When [updating to `5.0.13`](https://github.com/Homebrew/homebrew-core/pull/119298), it runs into openssl@1.1 linkage issue

```
  Broken dependencies:
    /opt/homebrew/opt/openssl@1.1/lib/libcrypto.1.1.dylib (openssl@1.1)
```

Reply:
@chenrui333 it's coming from erlang.
Compare https://github.com/Homebrew/homebrew-core/blob/master/Formula/erlang.rb#L33 vs https://github.com/Homebrew/homebrew-core/blob/master/Formula/erlang@24.rb#L26

We need to depend on the same openssl version as erlang - we package whatever we need from erlang runtime into our package, but since erlang formula does not link with openssl statically, we still need the same dynamic libraries in runtime.

Btw, just tested the latest version, and there is some issue. If you don't mind, I'll submit a PR to revert to erlang@24 and openssl@3 for a time being.

Reply:
@chenrui333 https://github.com/Homebrew/homebrew-core/pull/119313

Reply:
I checked `emqx` repo though, and they're testing against Erlang 25?

Reply:
Some tests are running on 25, but it is not officially supported yet, e.g. official packages are still built with 24 https://github.com/emqx/emqx/blob/master/.github/workflows/build_packages.yaml